ui/app/styles/components/ttl-picker.scss

```diff
@@ -11,6 +11,10 @@
   font-weight: bold;
 }
 
+.ttl-picker-form-field-error input {
```

Comment:
It seems like the pattern is to lump all the component styles together but since that means copying an existing style in this case I think it makes sense to move it. Good eye tracking this down!